pyfstat/make_sfts.py

```diff
@@ -742,6 +742,10 @@ def make_data(self, verbose=False):
         self.run_makefakedata()
 
     def run_makefakedata(self):
+        mfd_command_line = self.build_MFD_command_line()
+        self._execute_MFD(mfd_command_line)
+
+    def build_MFD_command_line(self):
```

Comment:
`_build_MFD_command_line`? Or is there a good reason for users to call it directly?

Reply:
Same as before, intended backdoor. But unnecessary, so removed in 2f31d5c.